.github/workflows/ci.yml

```diff
@@ -0,0 +1,51 @@
+name: CI
```

Comment:
Credit where credit is due. [This](https://www.pibit.nl/github/actions/rails/postgres/rspec/tutorial/example/2019/09/23/github-actions-with-rails-postgres-and-rspec/) article had all the code you see here, I just updated the ruby and postgres version to what i'm running